### PR TITLE
EHN: A divide-and-conquer, and brute-force algorithms for array inversions co…

### PR DESCRIPTION
**Pull request by maxwell-aladago**

### CODE REVIEW

**Comment by cclauss** [comment]
Please consider adding Python doctests to these three functions so that they will be tested by our Travis CI job.

**Comment by cclauss** [comment]
Travis CI tests are failing...  Let me know if you need help to decipher.

OPTIONAL:  Have you ever done Python type hints?  https://docs.python.org/3/library/typing.html

**Comment by maxwell-aladago** [comment]
> Travis CI tests are failing... Let me know if you need help to decipher.
> 
> OPTIONAL: Have you ever done Python type hints? https://docs.python.org/3/library/typing.html

Thank you, I was just looking at the travis build failure. it says something like no blank line after `>>>`. I don't know why that's the case but I am looking at it.

File: divide_and_conquer/inversions.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    Returns
+    -------
+    num_inversions: The total number of inversions in `arr`
+
+    Examples
+    ---------
+
+     >>> count_inversions_bf([1, 4, 2, 4, 1])
+     3
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

File: divide_and_conquer/inversions.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+     >>> count_inversions_bf([1, 1, 2, 4, 4]
+     0
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

File: divide_and_conquer/inversions.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    -------
+    C: a sorted copy of `arr`.
+    num_inversions: int, the total number of inversions in 'arr'
+
+    Examples
+    --------
+
+    >>> count_inversions_recursive([1, 4, 2, 4, 1])
+    [1, 1, 2, 4, 4], 3
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

File: divide_and_conquer/inversions.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    >>> count_inversions_recursive([1, 1, 2, 4, 4])
+    [1, 1, 2, 4, 4], 0
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

File: divide_and_conquer/inversions.py [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+    >>>> count_inversions_recursive([])
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Change 4 >>>> to 3 >>>

File: divide_and_conquer/inversions.py [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+    ------
+    R: array-like, a sorted array of the elements of `P` and `Q`
+    num_inversion: int, the number of inversions across `P` and `Q`
+
+    Examples
+    --------
+
+    >>> _count_cross_inversions([1, 2, 3], [0, 2, 5])
+    [0, 1, 2, 2, 3, 5], 4
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

File: divide_and_conquer/inversions.py [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+    >>> _count_cross_inversions([1, 2, 3], [3, 4, 5])
+    [1, 2, 3, 3, 4, 5], 0
+
```
Comment (cclauss) [inline divide_and_conquer/inversions.py]: Remove this blank line?

**Pull request merged**